cost/aws/instance_cloudwatch_utilization/AWS Inefficient Instance Utilization using CloudWatch.pt

```diff
@@ -184,14 +205,50 @@ datasource "ds_cloudwatch_windows_mem_usage" do
   end
 end
 
+# get AWS instance size map
+datasource "ds_aws_instance_size_map" do
+  request do
+    verb "GET"
+    host "raw.githubusercontent.com"
+    path "/rightscale/policy_templates/master/data/aws/instance_types.json"
+    header "User-Agent", "RS Policies"
+  end
+end
+
 datasource "ds_merged_metrics" do
   run_script $js_merged_metrics, $ds_cloudwatch_cpu_usage, $ds_cloudwatch_mem_usage, $ds_cloudwatch_windows_mem_usage
 end
 
+datasource "ds_inefficient_utilization_metrics" do
+  run_script $js_inefficient_utilization_metrics, $ds_aws_instance_size_map, $ds_merged_metrics, $param_avg_used_memory, $param_avg_cpu
+end
+
 ###############################################################################
 # Scripts
 ###############################################################################
 
+script "js_sorted_region", type: "javascript" do
+  result "sorted_region"
+  parameters "ds_regions_list"
+  code <<-EOS
+  var sorted_region=[];
+  _.each(ds_regions_list, function(region){
+    sorted_region.push(region);
+  })
+  sorted_region.sort(getSortOrder("region"));
+  function getSortOrder(prop){  
```

Comment:
use the understorejs sortby function instead of creating your own.

Reply:
Updated the code with sortby function